src/attributedlabel/src/NIAttributedLabel+Testing.h

```diff
@@ -0,0 +1,36 @@
+//
+// Copyright 2011-2014 NimbusKit
+// Originally created by Roger Chapman
```

Comment:
Does this header need updated?

Reply:
I copied this from the other header. I am assuming this is correct?